test/bit_stream.cpp

```diff
@@ -19,25 +19,10 @@
 #include "gmock/gmock.h"
 #include "util/bit_stream.h"
 
+namespace spvtools {
```

Comment:
Huh.  I never thought of putting an unnamed namespace inside a real namespace.

Reply:
It's nice because you can remove all the using statements below. Technically, I don't think the tests need to be wrapped in an anonymous namespace, but I don't think it hurts either.

Reply:
We like putting test code in anonymous namespace so there is no chance of having a dependency from the code being tested to the test code.